.travis.yml

```
@@ -4,13 +4,23 @@ language: dart
 jobs:
```

Try this configuration
```suggestion
dart:
  - stable
  - dev
  
jobs:
```

---

Both dart `stable` and `dev` are required to have a good score on pub. Web apps, cli apps and server app developers always use `stable` dart. Only flutter developers use `dev`. 😊

---

Yeah. Good suggestion.

---

I tried this configuration. But It didn't work properly when set both stable & dev globally.

I reverted the code changes. If you need to do the checks on dev env.  
Another PR is appreciated.